app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -57,10 +60,18 @@ object IntentValidator {
 
         when (intent.action) {
             Intent.ACTION_MAIN -> {
-                val dialParams = intent.extras?.getString(DIAL_PARAMS_KEY) ?: return null
-                if (dialParams.isNotEmpty()) {
+                val extras = intent.extras ?: return null
+
+                if (extras.keySet().contains(DIAL_PARAMS_KEY)) {
```

Comment:
This is a change in behavior: `dialParams` is no longer checked for `isNotEmpty`. Given this is unrelated to your change, can we change it back to keep the functionality the same?

It seems like there should have been a test for this. :)